wafer/pages/tests/test_menu.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+
+
+from wafer.menu import Menu
+from wafer.pages.models import Page, page_menus
+
+
+class PageMenusTests(TestCase):
+
+    def test_simple_nested_menu(self):
+        about = Page.objects.create(name='About', slug='about', include_in_menu=True)
+        contact = Page.objects.create(name='Contact', slug='contact', include_in_menu=True, parent=about)
+        menu = Menu([])
+        page_menus(menu)
+        self.assertEqual(menu.items, [
+            Menu.mk_menu("about", "About", sort_key=0, items=[
+                Menu.mk_item("Contact", contact.get_absolute_url(), sort_key=0)
+            ])
+        ])
+
+    def test_menu_ordering(self):
+        schedule = Page.objects.create(name='Schedule', slug='schedule', include_in_menu=True, menu_order=1)
+        about = Page.objects.create(name='About', slug='about', include_in_menu=True, menu_order=0)
+        menu = Menu([])
+        page_menus(menu)
+        self.assertEqual(menu.items, [
+            Menu.mk_item("Schedule", schedule.get_absolute_url(), sort_key=1),
+            Menu.mk_item("About", about.get_absolute_url(), sort_key=0),
```

Comment:
yes, thanks for pointing this out. since I dropped the default value, I modified this test to just check that the menu_order is passed along as sort_key to the menu.